### DISCUSSION TITLE
[INDEXER REQ] Karagarga

### DISCUSSION THREAD

**Discussion by xTalented** [autobrr/autobrr#1767] [Ideas]
Indexer name: Karagarga

    Short name: KG

    URL: https://karagarga.in

    Network name:  irc.brokensphere.net

    Network address: irc.learnbits.me

    Port: Port: 6667 or port 6670 for SSL connections

    SSL/TLS: Yes
    channels: #karagarga for general chat/questions (this channel is hidden, type /join #karagarga to access it).
#kg-help for account issues or other questions for staff.

**Comment by zze0s** [comment]
Hi! @xTalented Do they do announces on IRC? If yes, is there any links to autodl-irssi .tracker files in their wiki or can you provide examples as well as download links via for example RSS.

**Comment by rpatterson** [comment]
FYI, I am unable to get my ZNC configuration to connect to either `irc.brokensphere.net:+6697`, the server listed in [the docs](https://karagarga.in/chat.php), or `irc.learnbits.me:+6670`, the server listed in this issue's description. Also, I can't find anything that indicates that there is a channel in which new releases are announced.

**Comment by zsigisti** [comment]


**Reply by zze0s** [reply]
This is not the place to ask about invites.
